lib/cPanel/PublicAPI.pm

```diff
@@ -93,6 +94,13 @@ sub new {
         $self->{'ip'} = '127.0.0.1';
     }
 
+    $self->{'ua'} = HTTP::Tiny->new(
+        agent      => "cPanel::PublicAPI/$VERSION ",
+        verify_SSL => $self->{'ssl_verify_mode'},
+        keep_alive => $self->{'keepalive'},
+        timeout    => $self->{'timeout'} || 300,
```

Comment:
Minor: We already do `$self->{'timeout'} = $OPTS{'timeout'} || 300;` near the top of this function, so the `||` clause is redundant.